controllers/drplacementcontrol.go

```diff
@@ -1042,7 +1100,8 @@ func (d *DRPCInstance) runFinalSync(homeCluster string) (bool, error) {
 func (d *DRPCInstance) areMultipleVRGsPrimary() bool {
 	numOfPrimaries := 0
 
-	for _, vrg := range d.vrgs {
+	vrgs := d.fetchVRGsFromMClusterOrS3()
```

Comment:
This path is from Relocate where both cluster should be active. So checking for vrg using MCV would be sufficient.